code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -83,6 +83,30 @@
 		if("cry")
 			message = "<B>[comm_paygrade][src]</B> cries."
 
+		if("dab")
```

Comment:
Oh, yea, You should also add a return check if the mob is restrained/incapacitated or missing arms. The dabber can't dab if his arms have been disabled.